[KOGITO-9311] Suggested improvements for the Kogito Serverless Operator Documentation (#364)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc

kubectl logs deployment/kogito-serverless-operator-controller-manager -n kogito-serverless-operator-system -f
----

Once the operator is running, it will watch for instances of the {product_name} Custom Resources (CR). Using CRs, you can configure your {product_name} environment and define Workflows and builds to be handled by the operator.

> For the operator to correctly manage the CR instances, you must first install the {product_name} Custom Resource Definitions (CRDs). To check if the CRD definitions are correctly installed, try running:

Reviewer: ```suggestion
For the operator to correctly manage the CR instances, you must first install the {product_name} Custom Resource Definitions (CRDs). To check if the CRDs are correctly installed, try running:
```

== Uninstall the Operator
// TODO: this is super verbose for now because we don't have OLM/OperatorHub. This procedure should be replaced as soon as we have them. Or it should be renamed to "Uninstalling snapshot/local versions", so crazy users that don't rely on operatorhub/olm can also install/uninstall their instances.

> To uninstall the {operator_name}, first you should delete all the {product_name} Custom Resources (CR) instances managed by it. Once the CRs are deleted, you can then delete all resources created during the operator installation.

Reviewer 2: ```suggestion
To uninstall the {operator_name}, first, you must delete all the {product_name} Custom Resources (CR) instances managed by it. Once the CRs are deleted, you can delete all resources created during the operator installation.
```
